samples/test/two_step_test.py

```diff
@@ -102,6 +106,8 @@ def verify(
             pipeline_func=two_step_pipeline,
             verify_func=verify,
             mode=kfp.dsl.PipelineExecutionMode.V2_COMPATIBLE,
+            arguments={
+                kfp.dsl.ROOT_PARAMETER_NAME: ''},
         ),
```

Comment:
this TestCase was originally used to test using default GCS pipeline root, can we add a new test instead?
(you might want to have two separate verify funcs that share some logic, because one runs on MinIO and one runs on GCS)